proto/commonfate/control/account/account.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package commonfate.control.account.v1alpha1;
+
+service AccountService {
+  rpc GetAWSAccountDetail(GetAWSAccountDetailRequest) returns (GetAWSAccountDetailResponse) {} 
+}
+
+message GetAWSAccountDetailRequest {
+ string account_id = 1; 
+}
+
+message GetAWSAccountDetailResponse {
+    string account_id = 1;
```

Comment:
make this return the SSO region, and perhaps the SSO start URL. The SSO region we definitely have already, you might need to update our Terraform provider so that customers can add their start URLs.

(this was a pain previously in Granted because our APIs __didn't__ return this, so we had to ask the user client-side)